parsons/ngpvan/bulk_import.py

```diff
@@ -24,3 +28,127 @@ def get_bulk_import_resources(self):
         r = self.connection.get_request(f'bulkImportJobs/resources')
         logger.info(f'Found {len(r)} bulk import resources.')
         return r
+
+    def get_bulk_import_job(self, job_id):
+        """
+        Get a bulk import job status.
+
+        .. note::
```

Comment:
This is somewhat surprising to me - have you experienced this?

Reply:
Well, I was unable to recreate it, so I removed. Thanks for lifting this up.